charts/ccd-data-store-api/values.yaml

```diff
@@ -20,6 +20,7 @@ java:
     DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com
     DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-{{ .Values.global.environment }}
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified&sslmode=require"
+    DATA_STORE_DB_MAX_POOL_SIZE: 48
```

Comment:
Can we double check in AAT this has the same value as Prod?

Reply:
yes, it's 48 for aat as well in terraform.